functions/agentcore-proxy-lambda/main.py

```
@@ -0,0 +1,57 @@
+            payload=json.dumps(payload).encode("utf-8"),
+        )
+
+        result = json.loads(response["output"].read())
+
+        return {
+            "statusCode": 200,
+            "headers": CORS_HEADERS,
+            "body": json.dumps(result),
+        }
+
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)